src/components/accounts/ledger/SetupLedger.js

```diff
@@ -40,25 +41,32 @@ const SetupLedger = (props) => {
                 console.log('SetupLedger', DISABLE_CREATE_ACCOUNT, fundingOptions)
                 const publicKey = await dispatch(getLedgerPublicKey())
                 await setKeyMeta(publicKey, { type: 'ledger' })
+                Mixpanel.track("SR Set key meta")
 
                 if (DISABLE_CREATE_ACCOUNT && !fundingOptions) {
                     await dispatch(fundCreateAccountLedger(accountId, publicKey))
+                    Mixpanel.track("SR Fund create account ledger")
                     return
                 }
 
                 await dispatch(createNewAccount(accountId, fundingOptions, 'ledger', publicKey))
+                Mixpanel.track("SR Create new account ledger")
             } else {
                 await dispatch(addLedgerAccessKey())
+                Mixpanel.track("SR Add ledger access key")
             }
             await dispatch(refreshAccount())
         } catch(e) {
             setConnect('fail');
+            Mixpanel.track("SR Connect ledger failed", {error: e})
             throw e;
         } 
 
         if (isNew) {
+            Mixpanel.track("SR Go to profile of new account")
```

Comment:
I think it might be hard to distinguish between ledger, seed phrase, and other recovery methods with `SR`. We may want to include an identifier for each method

Reply:
sure, will change them to seperate